l3kernel/l3coffins.dtx

```diff
@@ -2463,7 +2463,7 @@
   {
     \@@_if_exist:NT #2
       {
-        #1 { LaTeX / kernel } { show-coffin }
+        #1 { LaTeX / coffins } { show-coffin }
           { \token_to_str:N #2 }
```

Comment:
Still have problems to understand why a few messages are `LaTeX/coffins`or `LaTeX/clist` but most others `\__kernel` ones

Reply:
These ones were I think done by @blefloch or @PhelypeOleinik, not me. I'll tidy them up so we are self-consistent.

Reply:
Turned out we didn't have `\__kernel_mg_show:nnnn`, which I'll sort first with a change on master then with one here.